src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Order.xml

```diff
@@ -221,6 +221,33 @@
                 </attribute>
             </itemOperation>
 
+            <itemOperation name="get_configuration">
+                <attribute name="method">GET</attribute>
+                <attribute name="controller">sylius.api.get_configuration_action</attribute>
+                <attribute name="path">/shop/orders/{id}/payments/{paymentId}/configuration</attribute>
```

Comment:
```suggestion
                <attribute name="path">/shop/orders/{id}/payments/{paymentId}/configuration</attribute>
```
Is order `id` used anywhere ?

Reply:
i guess it is, this was we secure endpoint for people with order token (we dont have it in payments)